node/node.go

```diff
@@ -269,6 +271,12 @@ func MakeFull(log logging.Logger, rootDir string, cfg config.Local, phonebookAdd
 	node.catchupService = catchup.MakeService(node.log, node.config, p2pNode, node.ledger, node.catchupBlockAuth, agreementLedger.UnmatchedPendingCertificates, node.lowPriorityCryptoVerificationPool)
 	node.txPoolSyncerService = rpcs.MakeTxSyncer(node.transactionPool, node.net, node.txHandler.SolicitedTxHandler(), time.Duration(cfg.TxSyncIntervalSeconds)*time.Second, time.Duration(cfg.TxSyncTimeoutSeconds)*time.Second, cfg.TxSyncServeResponseSize)
 
+	node.participationRegistry, err = ensureParticipationDB()
+	if err != nil {
+		log.Errorf("Cannot get participation metrics: %v", err)
```

Comment:
could you improve the text abit ? i.e. maybe something in the lines of `unable to initialize the participation database :  %v`